assets/scenes/load_scene_example.scn.ron

```diff
@@ -5,7 +5,7 @@
     ),
   },
   entities: {
-    0: (
+    4294967296: (
```

Comment:
Why use a random number and not `1`?

Reply:
It's not random :')
I've updated the first line in the PR description to be more clear.

Reply:
Ah, maybe add a comment so its more clear why that number is being used?

Reply:
Honestly the entire example is flawed and bound to be reworked when proper scene infrastructure rolls around, hopefully soon. Not sure if it's worth documenting at this point, you honestly just should not do serialization the way this example does, but I don't think deleting the example is an option